data/ships.txt

```diff
@@ -227,7 +227,8 @@ ship "Bactrian"
 	explode "large explosion" 45
 	explode "huge explosion" 30
 	"final explode" "final explosion large"
-	description "The Lionheart Bactrian is the last of the great city-ships, a design hearkening back to the early days of space colonization when a long-distance vessel needed to be a self-contained world, able to survive at times for weeks without encountering an inhabited planet. It is not only a freighter, but a carrier, and a capable warship either at short range or at a distance. Naturally, this versatility also makes it extremely expensive, and the Bactrian is not normally for sale to ordinary citizens who have not been vetted by the local government of the Deep."
+	description "The Lionheart Bactrian is the last of the great city-ships, a design hearkening back to the early days of space colonization when a long-distance vessel needed to be a self-contained world, able to survive at times for weeks without encountering an inhabited planet. It is not only a freighter, but a carrier, and a capable warship either at short range or at a distance. Naturally, this versatility also makes it extremely expensive."
+	description "In the past, the Bactrian was not normally for sale to ordinary citizens who had not been vetted by the local government of the Deep, but in recent times the Bactrian has been made available for all to purchase as newer designs are now more powerful than it."
```

Comment:
I would go with:

"In the past, the Bactrian was not normally for sale to ordinary citizens..."

Because @Pointedstick, that the citizens were vetted already excludes them from being called "ordinary". So it's either "normally for sale to citizens who were vetted," or "normally not for sale to ordinary citizens."